### DISCUSSION TITLE
ckan-scheming + filestore

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7234] [General]
Has anyone had any issues with using Ckan-scheming module + filestore uploads? 

both work individually but if I combine 1 CKAN instance with both i get 404 errors. 

![image](https://user-images.githubusercontent.com/115991393/203342062-bec7c5db-6c6c-4fd3-a1ab-be251af163d4.png)
if I comment out the below from Ckan.INI the site works fine, but as soon as I allow the filestore uploader i get the errors in the above image and cant work out why. 
ckan.storage_path = /var/lib/ckan/default

**Comment by wardi** [comment]
what version of ckan and ckanext-scheming are you using?

**Reply by ghost** [reply]
ckan 2.9,  

scheming - the latest from Master. 

being built in ubuntu 20.04

**Comment by wardi** [comment]
@JackBurns1984 I can't seem to reproduce this issue.

I tried testing with ckan dev-v2.9 branch commit aad4e73ebb10cf9eb3eef295942e094f4f875038 and ckanext-scheming master commit https://github.com/ckan/ckanext-scheming/commit/4e1dd0307c0366fb1152ff4bdafe0dddd7e47efd .
I tried with the `ckan_dataset.yaml` schema enabled and with no schemas enabled. Uploads work fine in both cases and I don't see any webasset errors in the server logs.

Not sure what else to try. @kowh-ai have you seen anything like this before?

**Reply by kowh-ai** [reply]
> 
> Not sure what else to try. @kowh-ai have you seen anything like this before?

Nope nothing - haven't had much to do with the Scheming plugin...

**Comment by ghost** [comment]
Hi @wardi   thanks for taking the time to look into this - very strange on my side then 

I will fire up a new vm with 20.04 and see if a fresh install works.

**Comment by ghost** [comment]
cheers both, must be me.  i'll write a fresh version over the weekend.
